skbio/stats/tests/test_composition.py

```diff
@@ -269,6 +335,92 @@ def test_centralize(self):
                             np.array([[2, 2, 6],
                                       [4, 4, 2]]))
 
+    def test_ancom_basic(self):
```

Comment:
Yeah, we can have all of those tests.

Really appreciate all of the feedback (seriously!) Keep the comments coming!